packages/shared/src/database/migrations/20200716154154_create-files.ts

```diff
@@ -0,0 +1,23 @@
+import * as Knex from 'knex'
+
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable('Files', function (table) {
+    table.increments('id')
+    table.string('owner').index().notNullable()
+    table.string('cid').unique().notNullable()
+    table.boolean('verified').index().defaultTo(false).notNullable()
+    table.integer('sizeBytes').unsigned().notNullable()
+    table.string('originalName')
+    table.string('encoding')
+    table.string('mimeType')
+    table.dateTime('expiresAt').index()
```

Comment:
This one is for the authentication right?

Reply:
Yep I will run a worker that will delete unverified files after this date